assets/settings/default.json

```diff
@@ -997,5 +997,6 @@
   //     ]
   //   }
   // ]
-  "ssh_connections": null
+  "ssh_connections": null,
+  "max_tabs": 0
```

Comment:
We shouldn't use `0` as a sentinel value.

The setting should be optional, where `None` means there is no limit and `Some(n)` means the limit is `n`.